src/clj/rems/auth/oidc.clj

```diff
@@ -162,9 +162,10 @@
                 user-info (when-let [url (:userinfo_endpoint oidc-configuration)]
                             (-> (http/get url {:headers {"Authorization" (str "Bearer " access-token)}})
                                 :body
-                                json/parse-string
-                                ga4gh/passport->researcher-status-by))
-                user (find-or-create-user! id-data user-info)]
+                                json/parse-string))
+                researcher-status (ga4gh/passport->researcher-status-by user-info)
+                id-data (merge id-data user-info)
```

Comment:
Perhaps we should merge `researcher-status` to here too? Then we could pass just the `id-data` which could perhaps be renamed to `user-data`?

Reply:
Alternatively we can merge the parsed `researcher-status` to `user-info` and pass both `id-data` and the full `user-info` to the `find-or-create-user!`.